README.md

```diff
@@ -1342,6 +1342,12 @@ Ensure all eligible online accounts have [multi-factor authentication](https://e
 
 Encrypt files locally before backing them up to external media or online services.
 
+You should follow the [3-2-1 backup model](https://www.cisa.gov/sites/default/files/publications/data_backup_options.pdf) as outlined by CISA.
```

Comment:
Let's not have this just become a catalog of links, especially to PDFs; explain terminology as it's defined.

Reply:
added an explanation